compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -281,118 +284,32 @@ class ValuePropagation : public TR::Optimization
    ValueConstraint *copyValueConstraints(ValueConstraints &valueConstraints);
 
 #if USE_TREES
-   class ValueConstraintHandler : public TREE_HANDLER<ValueConstraint>
+   class ValueConstraintHandler : public TREE_HANDLER <ValueConstraint>
       {
       public:
-      void setVP(TR::ValuePropagation *vp) {_vp = vp;}
```

Comment:
@mgaudet @0xdaryl @xliang6 can we have your input on the two choices available here and what should be done because we really want to get this finished so we can continue development work to further improve VP.